backend/src/models/requestsModel.ts

```diff
@@ -26,10 +26,21 @@ const requestCounter = () => {
   };
 };
 const validateRequest = async (request: CreateRequest): Promise<ModelResponseError | undefined> => {
-  if (request.title.trim() === '' || request.description.trim() === '') {
+  if (
+    [request.title, request.description, request.roomName].some(
+      (x) => x === undefined || typeof x !== 'string' || x.trim() === '',
+    )
+  ) {
     return { status: 400, message: 'Missing required fields.' };
   }
 
+  if (
+    request.approvers &&
+    (!Array.isArray(request.approvers) || request.approvers.some((x) => typeof x !== 'string'))
```

Comment:
does this need validation for if the approvers exist / have the required permission